package.json

```diff
@@ -47,6 +47,6 @@
     "webdriverio": "^4.6.2"
   },
   "devDependencies": {
-    "eslint": "2.9.0"
+    "eslint": "4.16.0"
```

Comment:
"^4.16.0" for compatible version. See [here](https://docs.npmjs.com/files/package.json#dependencies).

Reply:
Done.

I knew about the conflict, but wanted to get a pull request open so I could confirm that travis was working.